x-pack/plugin/eql/src/main/antlr/EqlBase.g4

```diff
@@ -205,12 +205,30 @@ LP: '(';
 RP: ')';
 PIPE: '|';
 
+fragment STRING_ESCAPE
+    : '\\' [btnfr"'\\]
+    ;
+
+fragment HEX_DIGIT
+    : [0-9abcdefABCDEF]
+    ;
+
+fragment UNICODE_ESCAPE
+    : '\\u' HEX_DIGIT HEX_DIGIT HEX_DIGIT HEX_DIGIT
```

Comment:
for consistency, can we drop the one and only support the `\u{xxxxxxxx}` style escape?

I think this might be one of those cases where less is more

Reply:
We discussed with @costin and we agreed that supporting the standard `\uXXXX` (without curly braces) which is the most common use case is a plus for user experience.

Reply:
I'm not sure I agree. @costin convinced me otherwise in https://github.com/elastic/eql/issues/8.
With `\uXXXX` it could easily mislead a user the types in `\u1f40`. That would be interpreted as `ὀd` not `🐍`.